content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,249 @@
+  icons, then clicking on the date.
+- :guilabel:`HR Responsible`: If there is a specific person in HR that is responsible for the
+  contract, select the person from the drop-down menu.
+- :guilabel:`Analytic Account`: This field allows a link between the contract and a specific
+  analytic account for accounting purposes.
+
+The contract details section allows for the addition and editing of a contract, and the ability to
+send the contract to the employee for approval and signatures.
+
+.. image:: contracts/contract-details.png
+   :align: center
+   :alt: Contract details in optional tabs for a new contract.
+
+- :guilabel:`Contract Template`: Select a pre-existing contract template from the drop-down menu.
+  Contract templates are typically created through the *Recrtuiment* application.
+- :guilabel:`New Contract Document Template`: Select a contract from the drop-down menu that will be
+  modified for this new employee contract.
+- :guilabel:`Contract Update Document Template`: Select a contract from the drop-down menu if the
+  employee has an existing contract that requires updating.
+- :guilabel:`Notes`: The notes field is a text field where any notes for the employee contract can
+  be entered for future reference.
+
+Modifying a contract
+~~~~~~~~~~~~~~~~~~~~
+
+Click the :guilabel:`Exteral Link` button at the end of each line to open the corresponding contract
+template and make any changes.
+
+.. image:: contracts/external-link.png
+   :align: center
+   :alt: Contract details in optional tabs for a new contract.
+
+A pop-up window wil appear with all the contract details. Modify the fields for the contract as
+needed.
+
+.. image:: contracts/modify-contract.png
+   :align: center
+   :alt: Edit the details for the contract.
+
+- :guilabel:`Tags`: Select any tags associated with the contract.
+- :guilabel:`Signed Document Workspace`: This is where the signatures will be housed. Choose a
+  pre-configured workspace or create a new one.
+- :guilabel:`Signed Document Tags`: Select or create any tags associated only with the signed
+  contract as opposed to the original unsigned contract.
+- :guilabel:`Redirect Link`: Enter a redirect link for the employee to access the contract. A
+  redirect link takes the user from one URL to another, in this case, to the newly updated contract
+  specifically written for them.
+- :guilabel:`Who can Sign`: Select either *All Users* or *On Invitation*.
+
+   - :guilabel:`All Users`: Any user in the organization can sign the contract.
+   - :guilabel:`On Invitation`: Only users selected in this field can sign the contract.
+
+- :guilabel:`Invited Users`: Select the person(s) that can sign the document.
+- :guilabel:`Document`: The attached document can be replaced by clicking the pencil icon. A pop-up
+  window will appear so another document can be selected for upload. The file
+  must be a PDF. To remove the document, click the clear icon (trash can icon).
```

Comment:
Maybe you want to modify how these icons are referred to as well (per our writers workshop meeting)?

Reply:
I added the emojis! I put the proper name that appears when you hover, which is Select and Clear (not intuitive descriptions, honestly).